### PR TITLE
Fix moveStalledJobsToWait: send failedReason to queueEvents

### PR DESCRIPTION
**Pull request by roggervalf**

### CODE REVIEW

File: src/commands/moveStalledJobsToWait-8.lua [hunk 1]
```diff
@@ -59,11 +60,13 @@ if (#stalling > 0) then
...
                                            1)
                 if (stalledCount > MAX_STALLED_JOB_COUNT) then
+                    local failedReason = "job stalled more than allowable limit" 
                     rcall("ZADD", KEYS[4], ARGV[3], jobId)
                     rcall("HSET", jobKey, "failedReason",
-                          "job stalled more than allowable limit")
+                          failedReason)
                     rcall("XADD", KEYS[8], "*", "event", "failed", "jobId",
-                          jobId, 'prev', 'active')
+                          jobId, 'prev', 'active', 'failedReason',
```
Comment (roggervalf) [inline src/commands/moveStalledJobsToWait-8.lua:68]: queueEvents has failed event and it expects a failedReason, here we are not sending any, so I add same failedReason

File: src/test/test_stalled_jobs.ts [hunk 2]
```diff
@@ -126,10 +129,28 @@ describe('stalled jobs', function() {
     await worker.close(true);
```
Comment (manast) [inline src/test/test_stalled_jobs.ts:130]: you could also assign the error message to a const here since you are using it in two places :).

**Review by manast — COMMENTED** [review]
good catch!

File: src/test/test_stalled_jobs.ts (hunk 2 above)
Comment (roggervalf) [reply src/test/test_stalled_jobs.ts:130]: sure, I missed that part xd

**Pull request merged**